### PR TITLE
Added notations to PGGD and other forms

### PR DESCRIPTION
**Pull request by DavidCrosswellGSQ**
I need notations for views in Redshift. Thanks, DC

### CODE REVIEW

File: vocabularies/georesources-report-types.ttl [hunk 1]
```diff
@@ -26,81 +26,81 @@ grt:pa-21A a skos:Concept ;
...
     skos:inScheme <http://linked.data.gov.au/def/georesource-report> ;
     skos:prefLabel "PGGD-02 Notice of completion, alteration or abandonment of petroleum well or bore"@en ;
-    skos:altLabel "Notice of completion, alteration or abandonment of petroleum well or bore"@en,
-        "PGGD-02"@en ;
+    skos:altLabel "Notice of completion, alteration or abandonment of petroleum well or bore"@en ;
+    skos:notation "PGGD-02"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
 
 grt:pggd-03 a skos:Concept ;
     skos:definition "A notice to inform the Department that hydraulic fracturing activities on a petroleum well or bore will performed."@en ;
```
Comment (talebib) [inline vocabularies/georesources-report-types.ttl:58]: ..... will be performed.

Comment (DavidCrosswellGSQ) [inline vocabularies/georesources-report-types.ttl:58]:
```suggestion
    skos:definition "A notice to inform the Department that hydraulic fracturing activities on a petroleum well or bore will be performed."@en ;

```

**Comment by KellyVance** [comment]
@DavidCrosswellGSQ is it just the notices you need notations for? I think some of the other reports that weren't part of the migration process or the original lodgement portal list had notations omitted, or never had notations to begin with e.g. articles, departmental drilling

**Comment by DavidCrosswellGSQ** [comment]
> @DavidCrosswellGSQ is it just the notices you need notations for?

@KellyVance - my immediate need is notations for the four PGGD forms, I fixed a few others while I was at it.
I think it is a low priority for notation for the others.

**Review by KellyVance — APPROVED** [review]
Shift of from altLabel to notations good.
File passes technical checks on multiple test tools.

**Review by talebib — APPROVED** [review]

**Pull request merged**